.travis.yml

```diff
@@ -1,10 +1,15 @@
 language: java
 
 jdk:
-  - oraclejdk7
+  - oraclejdk8
 
 before_install:
   - chmod +x gradlew
+
+# workaround for missing openjdk7 EC support
+# https://github.com/gradle/gradle/issues/2421#issuecomment-319916874
+install:
+ - JAVA_HOME=$(jdk_switcher home openjdk8) ./gradlew classes testClasses
```

Comment:
Is this necessary?

Reply:
No, this was a leftover from the openjdk7 EC workaround.